core/pom.xml

```diff
@@ -31,6 +31,12 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
Is this necessary? Doesn't spring boot test ship with Mockito?

Reply:
I have to retest but the version shipped with spring boot was to old to mock final classes.

Reply:
Yep I was right, the newes version is needed to mock `final` classes. In my case the `TransformationContext`